Sources/Recycling/PromotedCells/Subviews/FavoriteButton.swift

```diff
@@ -0,0 +1,39 @@
+import UIKit
+import FinniversKit
+
+class FavoriteButton: UIButton {
```

Comment:
Is there any difference between this and `IconButton` with the `.favorite` style?

Reply:
No difference, except the fact that I didn't know about this one😄 Thanks! Will replace and use this instead.